src/dashboard/cd-dashboard-news.vue

```diff
@@ -1,14 +1,77 @@
 <template>
   <div class="column">
     <div class="cd-dashboard-news">
-      <h3 class="cd-dashboard-news__header">Community News and Forum Updates</h3>
+      <h2 class="cd-dashboard-news__header">Community News and Forum Updates</h2>
+      <div class="cd-dashboard-news__posts" v-for="post in allPosts">
+        <span class="cd-dashboard-news__posts-left">
+          <p class="cd-dashboard-news__post-type">{{ post.type }}</p>
+          <p class="cd-dashboard-news__post-date">{{ post.date }}</p>
+        </span>
+        <span class="cd-dashboard-news__posts-right">
+          <h4 class="cd-dashboard-news__post-title">
+            <a class="cd-dashboard-news__post-title-link" :href="`${post.link}`">{{ post.title }}</a>
+          </h4>
+        </span>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
+  import NewsForumsService from './service';
+
   export default {
     name: 'cd-dashboard-news',
+    data() {
+      return {
+        news: null,
+        forums: null,
+      };
+    },
+    computed: {
+      allPosts() {
+        if (this.news && this.forums) {
+          const joinedPosts = ((this.news).map(post => ({
+            type: 'News',
+            date: (post.date.split('T')[0]).split('-').join(''),
+            link: post.link,
+            title: post.title.rendered,
+          })).concat((this.forums).map(post => ({
+            type: 'Forums',
+            date: (post.timestampISO.split('T')[0]).split('-').join(''),
+            link: `https://forums.coderdojo.com/topic/${post.slug}`,
+            title: post.title,
+          }))));
+
+          return this.sortPostsByDate(joinedPosts).splice(0, 6);
```

Comment:
If the `map`s are moved into their own properties as per my review in #173, then we could probably just move the sort into this computed property instead.

That said, there's nothing really wrong with having a method that does the sort separately either. I guess make the call when `allPosts` is simplified a bit?